subprojects/pv.wrap

```diff
@@ -0,0 +1,9 @@
+[wrap-file]
+directory = pv-1.8.5
+source_url = https://codeberg.org/a-j-wood/pv/releases/download/v1.8.5/pv-1.8.5.tar.gz
+source_hash = d22948d06be06a5be37336318de540a2215be10ab0163f8cd23a20149647b780
+source_filename = pv-1.8.5.tar.gz
+patch_directory = pv
+
+[provide]
+program_names =
```

Comment:
baffled how this works.

Reply:
I assume the context is muon failing on this?

Reply:
The context is rather that I don't understand the intention of defining this in a non-functional manner.

The fact that muon fails on this is interesting, but not as interesting as the question of "what was the goal here".

For example, the goal clearly wasn't to make meson work with `find_program('pv')`.